content/enterprise_features/e2e.md

```diff
@@ -307,6 +317,8 @@ By default, a Firefox test uses a new, blank profile for each test run. In order
 | VNC_VIEW_ONLY           | false                | Enable/Disable view-only mode                       |
 | NPM_TOKEN               |                      | NPM token to access npmjs.com registry              |
 | SAKULI_TRUSTED_CERT_DIR |                      | Directory containing custom certificates for import |
+| GIT_URL                 |                      | URL of git repository                               |
+| GIT_DIR                 | /headless/git-repository| Directory where the git repository is cloned     |
```

Comment:
Why would we need this configurable at all? Convention > Configuration. IMHO, it would be enough to clone the git repo somewhere we expect it during container runtime. Omitting `GIT_DIR` would also shorten the docker command and remove complexity from the customers resposibilities.